### PR TITLE
Flickr Widget: Adds option to open Flickr images in a new tab

### LINKED ISSUE #11405: Flickr Widget: Open image links in a new window
The Flickr widget can display a gallery of thumbnails linking to fullsize images on Flickr. These links currently open in the same window, taking visitors away from the site. It would be nice if there was an option to include `target="_blank"` on the thumbnail links to open them in a new browser window.

### PR DESCRIPTION
**Pull request by chickenn00dle**
Fixes #11405

#### Changes proposed in this Pull Request:
* Previously, Flickr widget images could only be opened in the same window which directs users away from the site owners' site. This PR adds an "Open images in new tab?" option to Flickr widget settings, allowing site owners to choose whether their users are directed away from their site or not.

#### Testing instructions:

1. Activate Jetpack Widgets
2. Add the Flickr (Jetpack) widget to a widget area of your choosing
3. Along with the other settings, check the "Open images in new tab?" checkbox
4. View the Flickr gallery in the front-end and open one of the images.
5. Uncheck the setting to verify images can open in the same window as well

#### Proposed changelog entry for your changes:
* Add option to open Flickr gallery images in new tabs

### CODE REVIEW

**Review by jeherve — CHANGES_REQUESTED** [review]
I would recommend setting up your IDE so it lints code as you work; that will avoid issues with the use of spaces instead of tabs in this PR, for example.

image

You can find some instructions here:
https://github.com/Automattic/jetpack/blob/master/docs/development-environment.md#use-php-codesniffer-and-eslint-to-make-sure-your-code-respects-coding-standards

Alternatively, if you run `yarn build` most the necessary tools should become available to you.

**Comment by chickenn00dle** [comment]
Ah, apologies!

Looks like I got ahead of myself there 😅

Followed up and made sure there were no issues with PHP CodeSniffer.

File: modules/widgets/flickr/form.php
Comment (jeherve) [inline modules/widgets/flickr/form.php]: I would place the checkbox before the label, I think it feels more natural.

**Review by jeherve — CHANGES_REQUESTED** [review]
This tests well. I would only have one minor recommendation for the look of the checkbox.

File: modules/widgets/flickr/form.php
Comment (chickenn00dle) [reply modules/widgets/flickr/form.php]: Good call. Actually reviewing some of the other checkboxes in other Jetpack widgets it looks like the checkbox and label text is nested within label tags. Made the change for this to be in-line with this pattern.

**Review by jeherve — APPROVED** [review]
This looks good to me! It should be good to merge after a second review!

**Pull request merged**
